docs/src/app/new/(content)/overview/about/page.mdx

```diff
@@ -0,0 +1,45 @@
+# About Base UI
+<Subtitle>An open-source React component library for building accessible user interfaces.</Subtitle>
+<Meta
+  name="description"
+  content="An overview of Base UI, providing information on its history, team, and goals."
+/>
+
+From the creators of Radix, MUI, and Floating UI, Base UI is an unstyled React component library for building accessible user interfaces. Our focus is on accessibility, performance, and developer experience. Our goal is to provide a complete set of open-source UI components, with a delightful developer experience, in a sustainable way.
+
+## Team
+
+- **Colm Tuite** (Radix)
+- **Vlad Moroz** (Radix)
+- **James Nelson** (Floating UI)
+- **Michal Dudak** (MUI)
+- **Marija Najdova** (MUI)
+- **Albert Yu** (MUI)
+
+## Features
+
+### Ship faster
+Whether you're crafting a component library or building a product feature, creating accessible UI components from native HTML elements can cost a lot of time and money. Base UI gives you the building blocks you need to assemble a styled component or complete user interface in a fraction of that time.
+
+### Own your CSS
+Wrestling with unwanted styles from a component library's CSS bundle can be time-consuming. Base UI components are unstyled, don't bundle CSS, don't prescribe a styling solution. You retain complete control over your application's CSS layer. Base UI is compatible with Tailwind, CSS Modules, plain CSS, CSS-in-JS, or any other styling engine you prefer.
```

Comment:
Same-ish, I'd prefer plain factual language that describes Base UI itself, like "Headless" or "Unstyled" instead of "Own your CSS"